md3-themes/MD3BluiDarkTheme.ts

```diff
@@ -48,10 +48,12 @@ export const MD3BluiDarkTheme = {
         onSurfaceDisabled: 'rgba(26, 27, 30, 0.38)',
         backdrop: 'rgba(45, 48, 56, 0.4)',
         // @TODO verify disabled color for dark theme
-        disabled: '#192024',
+        disabled: '#EBF1F4',
         // @TODO verify neutral30
+        // @TODO verify neutral80
         neutralVariant: {
             neutral30: '#414E54',
+            neutral80: '#BDCAD1',
             neutral100: '#FFFFFF',
         },
```

Comment:
This chunk should be taken out of theme configurations and be its own variable

Reply:
Do you want these 'neutral30', 'neutral80' and 'neutral100' variants be removed from the 'neutralVariant' and be it's own variables?

Reply:
correct. These should be coming from at least the "colors" package.